public/components/query_compare/search_result/search_components/search_configs/search_config.tsx

```diff
@@ -53,7 +84,9 @@ export const SearchConfig: FunctionComponent<SearchConfigProps> = ({
       selectIndex: '',
     }));
   };
-
+  
+  const documentIndex = queryNumber == 1? documentsIndexes1: documentsIndexes2
```

Comment:
We should use strict equality check
```suggestion
  const documentIndex = queryNumber === 1? documentsIndexes1: documentsIndexes2
```